src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
```

Comment:
The class Storage should be designed according to the encapsulation principle. The fruitStorage map is currently exposed as a public static field which allows it to be modified from anywhere in the code. This can lead to issues with data consistency and thread safety. Consider making fruitStorage private and providing methods to interact with it.